### ISSUE TITLE
Accessibility and performance in Lighthouse report are not satisfying

### ISSUE THREAD

**Issue by Asuka-Langley-Soryu** [animeshon/website-encyclopedia#6]
Whenever possible, performance and accessibility should be improved according to the Lighthouse report (Google Chrome).
